<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>721</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="11" column="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Scale colors</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="MyGLWidget" name="myGLWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <zorder>tabWidget</zorder>
       <zorder>tabWidget</zorder>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSlider" name="colorClampMinSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="drawingHedgehogsCheckbox">
       <property name="text">
        <string>Draw hedgehogs</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Dimensions</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="gridSizeLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Grid size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSlider" name="colorClampMaxSlider">
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Glyph scaling</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QSlider" name="gridSizeSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="drawGradientCheckbox">
       <property name="text">
        <string>Draw gradient</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="drawStreamLineCheckbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Stream lines</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QSlider" name="simulationTimestepSlider">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;timestep&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="sliderPosition">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QCheckBox" name="showGridCheckbox">
       <property name="text">
        <string>Show grid</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>VeclocityGlyphType</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Color clamp max</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fluid viscosity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="advancedOptionsButton">
       <property name="text">
        <string>Advanced options</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="dimensionsSpinbox">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="velocityGlyphType">
       <item>
        <property name="text">
         <string>hedgehogs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>arrows</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Timestep</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QSlider" name="hedgehogScalingSlider">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hedgehogscaling&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="drawingMatterCheckbox">
       <property name="text">
        <string>Draw matter</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QSlider" name="fluidViscositySlider">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;fluid viscosity&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="sliderPosition">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Color clamp min</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Color bands</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="applyColorTo">
       <item>
        <property name="text">
         <string>fluid density</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>|fluid velocity|</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>|force field|</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="scalarColoring">
       <item>
        <property name="text">
         <string>black&amp;white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rainbow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>heatmap</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="animationCheckbox">
       <property name="text">
        <string>Animate</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Number of glyps</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QSlider" name="numberOfGlyphsSlider">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>showAnimation(bool)</slot>
    <slot>drawMatter(bool)</slot>
    <slot>drawHedgehogs(bool)</slot>
    <slot>directionColoring(bool)</slot>
    <slot>timestep(int,int)</slot>
    <slot>hedgehogScaling(int,int)</slot>
    <slot>fluidViscosity(int,int)</slot>
    <slot>timestep(int)</slot>
    <slot>hedgehogScaling(int)</slot>
    <slot>fluidViscosity(int)</slot>
    <slot>drawHedgehogs()</slot>
    <slot>directionColoring()</slot>
    <slot>drawMatter()</slot>
    <slot>showAnimation()</slot>
    <slot>scalarColoring(QString)</slot>
    <slot>clampColorMin(int)</slot>
    <slot>clampColorMax(int)</slot>
    <slot>applyColoringToDataset(QString)</slot>
    <slot>setColorBands(int)</slot>
    <slot>setGlyphType(QString)</slot>
    <slot>scaleColor(bool)</slot>
    <slot>setDim(int)</slot>
    <slot>drawGrid(bool)</slot>
    <slot>setGridSize(int)</slot>
    <slot>drawDispersion(bool)</slot>
    <slot>setDrawGradient(bool)</slot>
    <slot>setDrawStreamline(bool)</slot>
    <slot>setNumberOfGlyphs(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>animationCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>showAnimation(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fluidViscositySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>fluidViscosity(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawingMatterCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>drawMatter(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hedgehogScalingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>hedgehogScaling(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simulationTimestepSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>timestep(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawingHedgehogsCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>drawHedgehogs(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalarColoring</sender>
   <signal>activated(QString)</signal>
   <receiver>myGLWidget</receiver>
   <slot>scalarColoring(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorClampMinSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>clampColorMin(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorClampMaxSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>clampColorMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyColorTo</sender>
   <signal>activated(QString)</signal>
   <receiver>myGLWidget</receiver>
   <slot>applyColoringToDataset(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setColorBands(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>871</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velocityGlyphType</sender>
   <signal>activated(QString)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setGlyphType(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>scaleColor(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dimensionsSpinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setDim(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dimensionsSpinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setDim(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showGridCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>drawGrid(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setGridSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGradientCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setDrawGradient(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawStreamLineCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setDrawStreamline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>gridSizeSlider</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>669</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>drawGradientCheckbox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>702</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>velocityGlyphType</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>871</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>colorClampMinSlider</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>showGridCheckbox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>880</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>colorClampMaxSlider</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>dimensionsSpinbox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_7</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>gridSizeLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>700</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedOptionsButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>numberOfGlyphsSlider</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>700</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberOfGlyphsSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setNumberOfGlyphs(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>793</x>
     <y>690</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
